<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibratorWidget</class>
 <widget class="QWidget" name="CalibratorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalibratorWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="startCalibrateButton">
       <property name="text">
        <string>Start Calibrate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopCalibrateButton">
       <property name="text">
        <string>Stop Calibrate</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
